source/java/product-guide/index.markdown

```diff
@@ -1743,8 +1743,8 @@ As we've [shown previously](#workflow-login-attempt), an account trying to login
 
 If you wish to change an account store's login priority for an application, you simply:
 
-1. Find the `accountStoreMapping` resource in the application's `accountStoreMappings` [collection](#collections) that reflects the `accountStore` that you wish to re-prioritize.
-2. Issue an update request to that `AccountStoreMapping`'s instance with a new `listIndex` value.
+1. Find the `applicationAccountStoreMapping` resource in the application's `accountStoreMappings` [collection](#collections) that reflects the `accountStore` that you wish to re-prioritize.
+2. Issue an update request to that `ApplicationAccountStoreMapping`'s instance with a new `listIndex` value.
```

Comment:
down here it says "by mapping `$accountStoreMapping` has". Could you please update it to "by mapping `accountStoreMapping` has". (note that I simply deleted the $))